### PR TITLE
Small updates to Dub Raycast Extension

### PR DESCRIPTION
**Pull request by steven-tey**
## Description

Updating the title, description, logo, and auth instructions

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by jfkisafk** [comment]
Hey @steven-tey , here is the command icon with Raycast masks applied if you want to use that:

dub-icon

**Comment by steven-tey** [comment]
@jfkisafk thank you so much, fixed both! 🙏

File: extensions/dub-link-shortener/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Dub Link Shortener Changelog
 
+## [Minor metadata updates] - 2024-07-12
```
Comment (jfkisafk) [inline extensions/dub-link-shortener/CHANGELOG.md:3]:
```suggestion
## [Minor metadata updates] - {PR_MERGE_DATE}
```

**Review by jfkisafk — APPROVED** [review]

**Comment by jfkisafk** [comment]
Hey @steven-tey just a minor comment, otherwise looks good to me.

**Comment by steven-tey** [comment]
@jfkisafk TIL about `{PR_MERGE_DATE}` – thank you!

**Review by jfkisafk — APPROVED** [review]

**Comment by jfkisafk** [comment]
Hello @mil3na can you please take a look when you find some time and assist in merging this? 🙏

**Review by mil3na — APPROVED** [review]
LGTM!

**Pull request merged**